Frends.Csv/Csv.cs

```diff
@@ -14,26 +14,24 @@ namespace Frends.Csv
 {
     public class Csv
     {
-
-
         /// <summary>
         /// Parse string csv content to a object. See https://github.com/FrendsPlatform/Frends.Csv
         /// </summary>
         /// <returns>Object { List&lt;List&lt;object&gt;&gt; Data, List&lt;string&gt; Headers, JToken ToJson(), string ToXml() } </returns>
         public static ParseResult Parse([PropertyTab] ParseInput input, [PropertyTab] ParseOption option)
         {
-
-            
             var configuration = new CsvConfiguration(new CultureInfo(option.CultureInfo))
             {
                 HasHeaderRecord = option.ContainsHeaderRow,
                 Delimiter = input.Delimiter,
-                TrimOptions = option.TrimOutput ? TrimOptions.None : TrimOptions.Trim, 
+                TrimOptions = option.TrimOutput ? TrimOptions.None : TrimOptions.Trim,
                 IgnoreBlankLines = option.SkipEmptyRows
             };
-           
-
 
+            if (option.TreatMissingFieldsAsNulls)
```

Comment:
Can you add a comment explaining the logic here? I am guessing that this is the delegate responsible for handling this situation and setting the delegate to null will make it return nulls for missing values. However, after 1 month it will already be not so obvious. It is always nice to comment unintuitive code places.